FabNESO/make_sweep_dir.py

```diff
@@ -70,10 +70,14 @@ def main() -> None:
         if not isinstance(parameter_dict, dict):
             msg = "Did not receive a dict as input for parameter_dict"
             raise ValueError(msg)
+        # If we define n_dirs in the command line argument, override any
+        # dict parameters we didn't assign n_dirs to
+        for _parameter, values in parameter_dict.items():
```

Comment:
```suggestion
        for _, values in parameter_dict.items():
```
`_` is conventionally used as dummy variable name for unused variable assignments

Reply:
With this one in particular I've just changed it to:
```python
for values in parameter_dict.values()
``` 
but your remark about the dummy variables is noted and taken onboard.